src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/module/OverviewMap.java

```diff
@@ -54,11 +54,10 @@
 	/**
 	 * The layers used within this OverviewMap.
 	 */
-	@ElementCollection
+	@ManyToMany
```

Comment:
`@ElementCollection` should only be used for basic or embedabble types, which was not correct here. It seems that not all of the config was respected in case of `@ElementCollection`